### ISSUE TITLE
Hello world gRPC example

### ISSUE THREAD

**Issue by SimonBiggs** [dicomrtai/dicomrtai#11]
Hi @pchlap,

After our chat today, I thought I'd jump on tonight and kick start things a little bit. So, I have set up a hello world client server gRPC protobuffer communication channel.

I have set up a server on DigitalOcean, and it is hosted at the address `api.dicomrt.ai`. There is an Ubuntu 20.04 machine running at that address that is running the following code:

https://github.com/dicomrtai/dicomrtai/blob/2a0d65b216774ba660c7d987d023b7ba9097031f/dicomrtai/_server.py#L24-L38

Essentially, it responds to all messages with `"Hello User!"`.

I have then made a client CLI. So, if you run:

```
pip install dicomrtai==0.2.2
dicomrt client
```

Your machine will run the following code:

https://github.com/dicomrtai/dicomrtai/blob/2a0d65b216774ba660c7d987d023b7ba9097031f/dicomrtai/_client.py#L23-L33

Which should then print the reply that the server sends back to you :slightly_smiling_face:.

---

Now just to have a hello-world demo set up for the file transfer, and have that be appropriately stored within something like an Amazon S3.

**Comment by pchlap** [comment]
Very nice @SimonBiggs! 

During development I can imagine it could be useful to test things out using localhost and a custom port? So having it configurable from the CLI might be useful for that?

Protocol Buffers is all brand new to me, do you have a recommended resource to learn about it? Or just head directly to: https://developers.google.com/protocol-buffers ?

**Comment by SimonBiggs** [comment]
> During development I can imagine it could be useful to test things out using localhost and a custom port? So having it configurable from the CLI might be useful for that?

Yup, good call. You can feel free to make those changes if you like, otherwise I'll likely jump on it again on Thursday night.
